clang/include/clang/Basic/LangOptions.def

```diff
@@ -275,6 +275,7 @@ LANGOPT(SYCLIsHost        , 1, 0, "SYCL host compilation")
 LANGOPT(SYCLAllowFuncPtr  , 1, 0, "Allow function pointers in SYCL device code")
 LANGOPT(SYCLStdLayoutKernelParams, 1, 0, "Enable standard layout requirement for SYCL kernel parameters")
 LANGOPT(SYCLUnnamedLambda , 1, 0, "Allow unnamed lambda SYCL kernels")
+LANGOPT(SYCLImplicitRPath , 1, 0, "Implicitely set RPath to SYCL library")
```

Comment:
There doesn't seem to be a use for this to be processed by the compiler as it is just a driver option.

Reply:
Right. I don't think we need this LangOpt

Reply:
Removed the LangOpt